db/collection.go

```diff
@@ -656,11 +739,11 @@ func getSchemaTypeProperties(jt *jsonschema.Type, defs jsonschema.Definitions) (
 	if jt.Ref != "" {
 		parts := strings.Split(jt.Ref, "/")
 		if len(parts) < 1 {
-			return nil, ErrInvalidCollectionSchema
+			return make(map[string]*jsonschema.Type), nil
```

Comment:
This should still be safe, because we _always_ check and add the _id as needed. So even if the schema doesn't call for it, all ThreadDB instances _will_ get an _id field. This makes it easier to support "empty" schemas.